src/client/permissions.py

```diff
@@ -0,0 +1,75 @@
+"""Client for Port permissions and RBAC operations."""
+
+from typing import Any
+
+from pyport import PortClient
+
+from src.utils import logger
+class PortPermissionsClient:
+    """Client for managing Port permissions and RBAC."""
+
+    def __init__(self, client: PortClient):
+        self._client = client
+
+    async def get_action_permissions(self, action_identifier: str) -> dict[str, Any]:
+        """Get permissions configuration for a specific action."""
+        logger.info(f"Getting permissions for action: {action_identifier}")
+        
+        try:
+            response = self._client.make_request("GET", f"actions/{action_identifier}")
```

Comment:
Here and in the PATCH requests, you used the wrong routes - you should use and pay attentio to the response and reqiuest schemas as there are issues there as well
https://docs.port.io/api-reference/update-an-actions-permissions
https://docs.port.io/api-reference/get-an-actions-permissions